apps/vr-tests-react-components/src/stories/Table.stories.tsx

```diff
@@ -27,6 +27,7 @@ import {
 import { Button } from '@fluentui/react-button';
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27953&commitId=fde7f26ea46db4db01e44ba22a27f254a1e7fa7b&env=prod)

✅ There was 2 screenshots added, 0 screenshots removed, 1940 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout flex.keyboard column resizing style.chromium.png|0| Added|
|Table layout table.keyboard column resizing style.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>